beta-src/src/state/game/game-api-slice.ts

```diff
@@ -529,7 +530,46 @@ const gameApiSlice = createSlice({
         }
       } else if (inProgress && method === "dblClick") {
         if (subsequentClicks.length) {
-          console.log("support move");
+          // user is trying to do a support move
+          const unitSupporting = ordersMeta[orderID];
+          const unitBeingSupported = subsequentClicks[0];
+          const terrEnum = Number(Territory[territoryName]);
+          const supportMoveMatch = unitSupporting.supportMoveChoices?.find(
+            ({ supportMoveTo: { name } }) =>
+              TerritoryMap[name].territory === terrEnum,
+          );
+          if (supportMoveMatch && supportMoveMatch.supportMoveFrom.length) {
+            const moveFromMatch = supportMoveMatch.supportMoveFrom.find(
+              ({ unitID: uID }) => {
+                return uID === unitBeingSupported.unitID;
```

Comment:
FYI - implicit return is added here on https://github.com/codazen/webDiplomacy/pull/104